examples/step-45/step-45.cc

```diff
@@ -14,417 +14,728 @@
  * ---------------------------------------------------------------------
 
  *
- * Author: Markus Buerg, University of Karlsruhe, 2010
+ * Author: Daniel Arndt, Matthias Maier, 2015
+ *
+ * Based on step-22 by Wolfgang Bangerth and Martin Kronbichler
  */
 
+// This example program is a slight modification of step-22 to demonstrate
+// the usage of periodic boundary conditions in deal.II. We thus omit to
+// discuss the majority of the source code and only comment on the parts
+// that deal with periodicity constraints. For the rest have a look at
+// step-22.
 
-// @sect3{Include files}
+// In order to implement periodic boundary conditions only two functions
+// have to be modified:
+// - StokesProblem<dim>::setup_dofs(): To populate a ConstraintMatrix
+//   object with periodicity constraints
+// - StokesProblem<dim>::run (): To supply a distributed triangulation with
+//   periodicity information.
 
-// The include files are already known. The one critical for the current
-// program is the one that contains the ConstraintMatrix in the
-// <code>lac/</code> directory:
-#include <deal.II/base/function.h>
-#include <deal.II/base/quadrature_lib.h>
 
-#include <deal.II/lac/constraint_matrix.h>
-#include <deal.II/lac/precondition.h>
+// @cond SKIP
```

Comment:
If you're going to skip things here, I'd prefer if you explicitly state this in the text above that you're going to skip some code, but that the full code is in the source directory.

That said, I think the code you're skipping here is in fact not the original step-22 code as you're using Trilinos. In my view, there would be no problem in not *skipping* code, but showing it and simply leaving it without any particular comments. This would at least make it easier for the reader to get the big picture.